Library/Feature.swift

```diff
@@ -2,12 +2,16 @@ import Foundation
 
 public enum Feature: String {
   case qualtrics = "ios_qualtrics"
+  case emailVerificationFlow = "ios_email_verification_flow"
+  case emailVerificationSkip = "ios_email_verification_skip"
 }
 
 extension Feature: CustomStringConvertible {
   public var description: String {
     switch self {
     case .qualtrics: return "Qualtrics"
+    case .emailVerificationFlow: return "Email Verification Flow"
+    case .emailVerificationSkip: return "Email Verification Skip"
```

Comment:
we don't need to localize this text, right?

Reply:
I don't see the other existing feature flag in the project localized so I think this is fine.